api/api/admin/media_report.py

```diff
@@ -857,6 +964,23 @@ def media_ids(self, obj):
     # Change view #
     ###############
 
+    change_form_template = "admin/api/media_decision/change_form.html"
+
+    def change_view(self, request, object_id, form_url="", extra_context=None):
+        # Expand the context based on the template's needs.
```

Comment:
```suggestion
```

Nit: this comment seems somewhat redundant or self-explanatory.

Reply:
It's been used in one more place so I added it for consistency. It also explains why we had to override the entire view just to add some more context for the template.